src/ZPublisher/tests/test_WSGIPublisher.py

```diff
@@ -784,6 +784,48 @@ def test_set_REMOTE_USER_environ(self):
         self._callFUT(environ, start_response, _publish)
         self.assertFalse('REMOTE_USER' in environ)
 
+    def test_webdav_source_port(self):
+        from ZPublisher import WSGIPublisher
+        start_response = DummyCallable()
+        _response = DummyResponse()
+        _publish = DummyCallable()
+        _publish._result = _response
+
+        # WebDAV source port not configured
+        environ = self._makeEnviron(PATH_INFO='/test')
+        self.assertNotIn('WEBDAV_SOURCE_PORT', environ)
+        self.assertEqual(WSGIPublisher._WEBDAV_SOURCE_PORT, 0)
+        self._callFUT(environ, start_response, _publish)
+        self.assertNotIn('WEBDAV_SOURCE_PORT', environ)
+        self.assertEqual(environ['PATH_INFO'], '/test')
+
+        # Configuring the port
+        WSGIPublisher.set_webdav_source_port(9800)
```

Comment:
As this statement changes a global variable there should be a try-finally-clause to reset it at the end of the test or in teardown.

Reply:
A added a cleanup step at the end of the test